core/quality/mysterium_morqa.go

```diff
@@ -472,6 +472,28 @@ func (m *MysteriumMORQA) ProviderTransferredDataSeries(id identity.Identity, ran
 	return data, nil
 }
 
+// ProviderServiceEarnings fetch earnings per service type.
+func (m *MysteriumMORQA) ProviderServiceEarnings(id identity.Identity) (node.EarningsPerService, error) {
+	var data node.EarningsPerService
+	request, err := requests.NewSignedGetRequest(m.baseURL, "provider/service-earnings", m.signer(id))
+	if err != nil {
+		return data, err
+	}
+
+	response, err := m.client.Do(request)
+	if err != nil {
+		return data, errors.Wrap(err, "failed to request service earnings")
```

Comment:
Let's not use `errors.Wrap` for errors everywhere, we were trying to migrate from it into `fmt.Errorf("my context: %w", err)`

https://github.com/mysteriumnetwork/devbook/tree/master/golang/error-handling